### DISCUSSION TITLE
이슈에 버그리포트에서 라벨을 저희가 달수가 없는데요?

### DISCUSSION THREAD

**Discussion by ted002** [naverz/zepeto-studio-kor#719] [5. Q&A]
의도하신건가요 아니면 실수하신건가요?

이번에 영상녹화 관해서 이슈리포트 하는데 아무리봐도 버그라벨을 제가 달수가 없어서요

**Comment by smilejsu82** [comment]
저도 안되든데요

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 라벨 세팅 관련해서 불편을 드려 죄송합니다.
템플릿에서 라벨 필수 항목은 제거했습니다.
감사합니다.
